code/obj/item/misc_weapons.dm

```diff
@@ -2130,6 +2130,62 @@ obj/item/whetstone
 #undef HALB_MED_STAMCOST
 #undef HALB_LIGHT_STAMCOST
 
+/obj/item/crashaxe
+	name = "crash axe"
+	desc = "A lightweight utility-survival axe designed to cut through the thin skin of pod hulls. Works just as well on bothersome crewmembers."
+	icon = 'icons/obj/items/weapons.dmi'
+	icon_state = "crashaxe"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	hitsound ='sound/impact_sounds/coconut_break.ogg' //shamelessly re-using this sound idea from my halberd
+
+	health = 5
+	tool_flags = TOOL_CUTTING | TOOL_CHOPPING
+
+	w_class = W_CLASS_SMALL
+	hit_type = DAMAGE_CUT
+	leaves_slash_wound = TRUE //yeah prolly
```

Comment:
Should this be on both sides?

Reply:
I thought about having it toggle between cut and stab, but that seemed a little convoluted